frontend/src/pages/result/VoteResult.tsx

```diff
@@ -1,13 +1,81 @@
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
 import DisplayMenus from "../../components/general/DisplayMenus";
+import VoteTimeTable from "../../components/result/VoteTimeTable";
+import VoteRestaurantTable from "../../components/result/VoteRestaurantTable";
+import {DataCompleteDateInfos} from "../../service/tastydate-api-service";
 
-export default function VoteResult() {
+interface TabPanelProps {
+    children?: React.ReactNode;
+    index: number;
+    value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+    const { children, value, index, ...other } = props;
+
+    return (
+        <div
+            role="tabpanel"
+            hidden={value !== index}
+            id={`simple-tabpanel-${index}`}
+            aria-labelledby={`simple-tab-${index}`}
+            {...other}
+        >
+            {value === index && (
+                <Box sx={{ p: 3 }}>
+                    <Typography>{children}</Typography>
+                </Box>
+            )}
+        </div>
+    );
+}
+
+function a11yProps(index: number) {
+    return {
+        id: `simple-tab-${index}`,
+        'aria-controls': `simple-tabpanel-${index}`,
+    };
+}
```

Comment:
I would inline this or extract into own file

Reply:
I created an own file! It definitely looks better now!